src/picai_prep/preprocessing.py

```diff
@@ -118,16 +118,35 @@ def resample_img(
 
 def crop_or_pad(
     image: "Union[sitk.Image, npt.NDArray[Any]]",
-    size: Iterable[int] = (64, 64, 64)
+    size: Optional[Iterable[int]] = (20, 256, 256),
+    physical_size: Optional[Iterable[float]] = None,
 ) -> "Union[sitk.Image, npt.NDArray[Any]]":
     """
     Resize image by cropping and/or padding
+
+    Parameters:
+    - image: image to be resized (sitk.Image or numpy.ndarray)
+    - size: target size in voxels (z, y, x)
+    - physical_size: target size in mm (z, y, x)
+
+    Either size or physical_size must be provided.
+
+    Returns:
+    - resized image (same type as input)
     """
     # input conversion and verification
+    if physical_size is not None:
+        # convert physical size to voxel size (only supported for SimpleITK)
+        assert isinstance(image, sitk.Image), "Crop/padding by physical size is only supported for SimpleITK images."
```

Comment:
assert should not be used in production code. use if/raise instead.

Reply:
I agree. Made a new commit to tackle this. Also, added a new function `crop` where a maximum size can be specified. I need this functionality, do you think this is the right place to store this function? (as it is not used inside the picai_prep module itself, but does pertain to preprocessing).